pkg/reporter/format.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/google/osv-scanner/pkg/models"
 )
 
-var format = []string{"table", "vertical", "json", "markdown", "sarif", "gh-annotations", "cyclonedx-1-4", "cyclonedx-1-5"}
+var format = []string{"html", "table", "vertical", "json", "markdown", "sarif", "gh-annotations", "cyclonedx-1-4", "cyclonedx-1-5"}
```

Comment:
Is there any particular order for format here? if not, shall we order alphabetically?

Reply:
Removed html from help command for now.